ebu_tt_live/carriage/filesystem.py

```diff
@@ -56,17 +56,33 @@ class FilesystemProducerImpl(AbstractProducerCarriage):
 
     _manifest_path = None
     _dirpath = None
-    _manifest_content = None
```

Comment:
This line was removed. On purpose?

Reply:
Yes, `_manifest_content` wasn't ever used anywhere as far as I could see.